src/primitives/transaction.h

```diff
@@ -22,6 +22,9 @@
 #include <tuple>
 #include <utility>
 #include <vector>
+#if __cplusplus >= 202002L
```

Comment:
Not a fan of adding stuff like this just to appease a tool, which happens to be getting run under a non-default compilation mode (`--enable-c++20`).

Reply:
See 89bcb1a856a563013062c2c129012498e6b8b1f9

Reply:
I don't think we want to just drop the option without adding it to a different CI (and not a Windows build).

Reply:
> I don't think we want to just drop the option without adding it to a different CI (and not a Windows build).

Agree. Also https://github.com/bitcoin/bitcoin/pull/25528#discussion_r1058955188

Getting ready a proposal.

Reply:
> Getting ready a proposal.

#26770

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/26763#issuecomment-1367571160).